### PR TITLE
1.0.2i changed the way COMP_METHOD is exported if NO_COMP is set

### PR DESCRIPTION
**Pull request by reaperhulk**

### CODE REVIEW

File: src/_cffi_src/openssl/ssl.py [hunk 1]
```diff
@@ -664,7 +664,8 @@
 #endif
 
 
-#if defined(OPENSSL_NO_COMP) || defined(LIBRESSL_VERSION_NUMBER)
+#if (defined(OPENSSL_NO_COMP) && CRYPTOGRAPHY_OPENSSL_LESS_THAN_102I) || \
```
Comment (alex) [inline src/_cffi_src/openssl/ssl.py]: Please add a comment.

**Comment by alex** [comment]
`CRYPTOGRAPHY_OPENSSL_LESS_THAN_102I` needs to be exposed in `cryptography.py`, not just declared

**Comment by raycoll** [comment]
Any chance this will be backported to the older versions: 1.2.x, 1.3.x, 1.5.x?

**Comment by reaperhulk** [comment]
cryptography has historically chosen to only support the latest version. Is there a reason you need to compile against a recent OpenSSL but can't run the latest cryptography?

**Pull request merged**